pg_duckdb/run.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+TRIES=3
+
+cat queries.sql | while read -r query; do
+    sync
+    echo 3 | sudo tee /proc/sys/vm/drop_caches
+
+    echo "$query";
+    for i in $(seq 1 $TRIES); do
+        psql postgres://postgres:duckdb@localhost:5432/postgres -t -c '\timing' -c "$query" | grep 'Time'
+    done;
```

Comment:
Opening and closing the postgres connection for each query has significant overhead when connecting to MotherDuck, because each postgres backend needs to open its own connection to motherduck. Writing a Python script instead might be preferable, otherwise you can do something like this.

```suggestion
    echo "$query"
    (
        echo '\timing'
        yes "$query" | head -n $TRIES
    ) | psql postgres://postgres:duckdb@localhost:5432/postgres -t | grep 'Time'
```

Reply:
Other than that, for MotherDuck it's important that the machine that runs the benchmark is in AWS us-east-1 to get the lowest latency to MotherDuck.